lib/cfn-nag/custom_rules/AmplifyBranchBasicAuthConfigPasswordRule.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'password_base_rule'
+
+class AmplifyBranchBasicAuthConfigPasswordRule < PasswordBaseRule
+  def rule_text
+    'Amplify Branch BasicAuthConfig Password must not be a plaintext ' \
+    'string or a Ref to a NoEcho Parameter with a Default value.' \
+  end
+
+  def rule_type
+    Violation::FAILING_VIOLATION
+  end
+
+  def rule_id
+    'F60'
+  end
+
+  def resource_type
+    'AWS::Amplify::Branch'
```

Comment:
Amplify::App also has this same auth config

Reply:
Yep, and it was already merged into master in a previous commit and PR, https://github.com/stelligent/cfn_nag/commit/eebe8f32904002e7a0e20ac040960fbd6d22f992